pkg/kubelet/server.go

```diff
@@ -346,6 +352,18 @@ func (s *Server) InstallDebuggingHandlers() {
 	s.restfulCont.Add(ws)
 }
 
+// InstallApiHandlers registers the HTTP request patterns that serve the kubelet versioned api.
+func (s *Server) InstallApiHandlers() {
+	ws := new(restful.WebService).
+		Path("/api/v1/").
+		Consumes(restful.MIME_JSON).
+		Produces(restful.MIME_JSON)
+
+	ws.Route(ws.GET("rawMetrics").To(handlers.RawMetricsHandler(s.host, s.host.GetStatusManager())))
```

Comment:
The term `metrics` is already overloaded. We thought `rawMetrics` would convey the intended meaning. 
In the future, we will have `derivedMetrics` which will be distributions. 
I'd prefer not using `metrics`. 
Any alternative suggestions @jimmidyson ?

> `/metrics/nodes[/nodeId]`

Since this is at the node level, what will `nodeId` be used for?

> /metrics/pods
/namespaces/<namespace>/metrics/pods[/podId]

+1 for this split.